packages/frontend-design-poc/src/components/PageLayout/PageLayout.tsx

```diff
@@ -2,20 +2,14 @@ import React from 'react';
 import { Outlet } from 'react-router-dom';
 import { Footer, Header, Sidebar } from '..';
 import styles from './pageLayout.module.css';
+import cx from 'classnames';
 
 export const PageLayout: React.FC = () => {
   const [companyName, setCompanyName] = React.useState<string>('Aker Solutions AS');
   const isCompany = !!companyName;
 
-  const appStyle: React.CSSProperties & { [key: string]: string } = {
-    '--background-color': isCompany ? '#E9F5FF' : '#EAF7EF',
-    '--button-color': isCompany ? '#111D46' : '#084826',
-    '--avatar-color': '#111D46',
-    '--action-important': '#E02E49',
-  };
-
   return (
-    <div style={appStyle} className="app">
+    <div className={cx({ isCompany })}>
```

Comment:
cx er vel ikke nødvendig her

Reply:
Shouldn't it be something like `className={isCompany ? styles.isCompany : ''}`? Now it'll set a `className` to be `true`.

Reply:
Jepp kan gjøre det manuelt på den måten for å slippe å importere 'classnames', men det gjør det samme. Det jeg hadde i koden (cx({ isCompany })) setter classname "isCompany" hvis isCompany verdien er true, er en shorthand.